Mage.Tests/src/test/java/org/mage/test/cards/designations/StartYourEnginesTest.java

```diff
@@ -0,0 +1,148 @@
+package org.mage.test.cards.designations;
+
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import mage.players.Player;
+import org.junit.Assert;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestPlayerBase;
+
+/**
+ * @author TheElk801
+ */
+public class StartYourEnginesTest extends CardTestPlayerBase {
+
+    private static final String sarcophagus = "Walking Sarcophagus";
+
+    private void assertSpeed(Player player, int speed) {
+        Assert.assertEquals(player.getName() + " speed should be " + speed, speed, player.getSpeed());
+    }
+
+    @Test
+    public void testRegular() {
+        setStrictChooseMode(true);
+        setStopAt(1, PhaseStep.POSTCOMBAT_MAIN);
+        execute();
+
+        assertSpeed(playerA, 0);
+    }
+
+    @Test
+    public void testSpeed1() {
```

Comment:
For info: there are must be init speed on sarcophagus's control change. There aren't any tests for it, but current static ability implementation looks compatible with such use cases, so all fine.

Reply:
yeah I'll add a test for that